internal/unix/types_other.go

```diff
@@ -48,6 +48,8 @@ const (
 	RLIM_INFINITY            = 0x7fffffffffffffff
 	RLIMIT_MEMLOCK           = 8
 	BPF_STATS_RUN_TIME       = 0
+	PERF_RECORD_LOST         = 2
```

Comment:
Isn't `PERF_RECORD_*` Linux-specific? Also, isn't the perf API Linux-specific? I see the BSD's have PMC instead of perf. If so, why do we put the constants here behind build tags but not the perf package itself? For anything non-Linux, we should have stubs that don't refer to Linux symbols, which would allow us to remove unnecessary duplicates here.

Same goes for epoll constants etc.

cc @lmb

Reply:
We can just use zero for all these constants.